src/snake.h

```diff
@@ -46,6 +49,7 @@ namespace snake
         void getInput();
         void updateSnakePos();
         void calcNewSnakePos();
+        void increaseSnakeSpeed();
 
         /* fixes */
         inline void normaliseMovementSpeed() const;
```

Comment:
isn't necessary. if you think it is, you should make an on off switch for it, because if I am debugging or running it, up/down is slower. if I remove it it is normal speed.

Reply:
It's added!